addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -68,6 +68,30 @@ export class CallParticipantCard extends Component {
         );
     }
 
+    get isRemoteVideo() {
+        if (!this.rtcSession) {
+            return false;
+        }
+        return (
+            this.rtc.isRemote &&
+            (this.rtcSession.is_screen_sharing_on || this.rtcSession.is_camera_on)
+        );
+    }
+
+    get showLiveLabel() {
+        if (this.props.isSidebarItem) {
+            return false;
+        }
+        return (
+            this.isRemoteVideo ||
+            (this.rtcSession?.is_screen_sharing_on && !this.props.minimized && !this.isOfActiveCall)
+        );
+    }
```

Comment:
The "live" is reserved for the screen-sharing, not the camera.

For showing that there's a card with camera placeholder in non-main tabs, I would use the `.fa-video-camera` icon